group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupConfig.java

```diff
@@ -237,4 +266,20 @@ public int shareHeartbeatIntervalMs() {
     public int shareRecordLockDurationMs() {
         return shareRecordLockDurationMs;
     }
+
+    /**
+     * The share group auto offset reset strategy.
+     */
+    public ShareGroupAutoOffsetReset shareAutoOffsetReset() {
+        return ShareGroupAutoOffsetReset.valueOf(shareAutoOffsetReset.toUpperCase(Locale.ROOT));
+    }
+
+    public enum ShareGroupAutoOffsetReset {
+        LATEST, EARLIEST, NONE;
```

Comment:
This doesn't really have the value "NONE" does it? The KIP mentions only LATEST and EARLIEST I think.